discordbot/commands/explain.py

```diff
@@ -71,6 +71,15 @@ async def explain(ctx: MtgContext, *, thing: Optional[str]) -> None:
                 'Current League': fetcher.decksite_url('/league/current/')
             }
         ),
+        'netdecking': (
+            """
+            If you are having trouble coming up with an original decklist, feel free to start by learning from the best!
+            You can find the best tournament winning decks in the link below. Sort by stars to get decks that won tournaments or did well in tournaments and leagues.
+            """,
+            {
+                'Decks': fetcher.decksite_url('/decks/'),
```

Comment:
If we do go with the shorter text it might be nice to call this "Decklists" not "Decks"?

Reply:
Got it.